### ISSUE TITLE
Returing user: Pressing 0 to undergo training once again triggers invalid input message

### ISSUE THREAD

**Issue by Madhuvanthi-A** [janastu/pyGSM#7]

**Comment by sankethkumar** [comment]
@Madhuvanthi-A did you check with 3?
what do you suggest? to press 3 or 0 for training once again?

**Comment by salus-sage** [comment]
Whatever it is Shud be low priority, these r optimization probs, and
saverio can get back on this later.

**Comment by Madhuvanthi-A** [comment]
@sankethkumar Yes, I tried with 3 and other inputs as well. By pressing 3 caller is taken to training level. Once caller have undergone training session, survey is resumed.
